src/main/java/blackjack/domain/Gamer.java

```diff
@@ -0,0 +1,97 @@
+package blackjack.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public abstract class Gamer {
+
+    public static final int NUM_INIT_CARD = 2;
+    private static final String SPACE = " ";
+    private static final String ERROR_NAME_SPACE = "이름에 공백이 포함됩니다.";
+    private static final int ACE_BONUS = 10;
+    private static final int AVAILABLE_ACE_BONUS = 11;
+    public static final int HIGHEST_POINT = 21;
+    private static final String COMMA = ", ";
+    private final String name;
+    private final List<Card> cards = new ArrayList<>();
+
+    protected Gamer(String name) {
+        validateSpace(name);
+        this.name = name;
+    }
+
+    private static void validateSpace(String name) {
+        if (name.contains(SPACE)) {
+            throw new IllegalArgumentException(ERROR_NAME_SPACE);
+        }
+    }
+
+    protected void receiveCard(Card card) {
+        cards.add(card);
+    }
+
+    protected int calculateJudgingPoint() {
+        return cards.stream().map(Card::givePoint).reduce(0, Integer::sum);
```

Comment:
아래처럼 줄을 바꿔주면 가독성이 더 좋을 것 같아요 :) 
```suggestion
        return cards.stream()
                .map(Card::givePoint)
                .reduce(0, Integer::sum);
```

Reply:
변경했습니다